webroot/results/person.php

```diff
@@ -66,6 +66,17 @@ function showBody () {
   if( $picture )
     echo "<center><img class='person' src='$picture' /></center>";
 
+  #--- Show the In Memoriam if any.
+  $inMemoriamArray = array(
+    "2008COUR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=2028",
```

Comment:
Just curious: do you know what the `f=9` is doing here? It looks like <https://www.worldcubeassociation.org/forum/viewtopic.php?t=2028> works as well.